web/libs/editor/src/tags/object/RichText/view.jsx

```diff
@@ -362,7 +362,7 @@ class RichTextPieceView extends Component {
   render() {
     const { item } = this.props;
 
-    if (!item._value) return null;
+    if (!item._value === null) return null;
```

Comment:
would it be safer to go with `if (!item.value && item.value !== '')` to change the behavior a bit less?